src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -66,6 +67,7 @@ protected function configure(): void
                 new InputOption('memory-limit', 'm', InputOption::VALUE_REQUIRED, 'The memory limit the worker can consume'),
                 new InputOption('time-limit', 't', InputOption::VALUE_REQUIRED, 'The time limit in seconds the worker can run'),
                 new InputOption('bus', 'b', InputOption::VALUE_REQUIRED, 'Name of the bus to which received messages should be dispatched', $defaultBusName),
+                new InputOption('silent', 's', InputOption::VALUE_REQUIRED, 'Does not stop when an exception is thrown', false),
```

Comment:
This change introduces a 'do not die if a message handler throws an exception' option.

An `silent` option suggests that the command is otherwise non-silent and that the addition of this option addresses existing non-silent behaviour. From what I've seen in my usage of the messenger component, `messenger:consume-messages` is currently pretty good at being silent.

It doesn't seem that this option relates to the silencing of existing non-silent behaviour.

Naming the option `do-not-die-if-a-message-handler-throws-an-exception` is closer to describing what the option does. Obviously this is not the best choice as it is a little on the verbose side.

Can this option be re-named to better convey what is being addressed?

Reply:
So `--force?` Silent to me sounds confusing with existing `--quiet|-q`

Reply:
I'm not sure about `--force`. It's certainly not as confusing as `silent` could be with respect to the existing `--quiet` but it still lacks a little descriptiveness.

I'm considering this from the perspective of seeing the command in use and being able to understand what will happen based on the options being provided.

`messenger:consume-messages --force` doesn't tell me much about what is being forced, particularly given how `--force` is often used to say 'do this anyway despite the potential for destructive irreversible changes`.

`messenger:consume-messages --swallow-exceptions` comes closer to conveying what is actually happening. I don't like this as it is too implementation-specific (I'm assuming the possibility of implementing the desired behaviour without having to swallow exceptions).

`messenger:consume-messages --naming-is-hard` ...

Reply:
> messenger:consume-messages --force doesn't tell me much about what is being forced

It does to me: "force the consumption of messages".

Running it with -vvv you'd see the logs.

Reply:
Cool, thanks for the clarification.

"force the consumption of messages" in this context expands to "force the consumption of messages even if an exception is thrown by a message handler".

This could be expressed in a more generic form of "force the consumption of messages in the case of X". 

If X is only ever "if an exception is thrown by a message handler" then `--force` seems fine to me now that I understand where you're coming from.

On the other hand, if we would ever need to force the consumption of messages for a reason other than due to an exception being thrown by a message handler, `--force` is probably too vague.

Reply:
So i try with `force` option?

Reply:
@roukmoute I'm not certain that `force` is definitely what is needed. It's better than `silent`.

My feeling is that it would be better as `force-<something descriptive>` but, due to naming being hard, I haven't any useful suggestions.